app/templates/partials/questions/repeatinganswer.html

```diff
@@ -45,7 +45,7 @@ <h3 class="neptune">{{ _('Person ') }}<span class="js-household-loopindex">{{ lo
   </div>
 
   <div class="question__actions u-mt-xs">
-    <button class="btn btn--secondary btn--small u-wa--@xs js-household-btn" type="submit" name="action[add_answer]">
+    <button class="btn btn--secondary btn--small u-wa--@xs js-household-btn u-mb-m" type="submit" name="action[add_answer]">
       {{ _('Add another person') }}
       <span class="u-vh">. Selecting this will add a new set of fields to enter another person</span>
```

Comment:
Not related to this PR. We need to mark all static content like this for translation. Feel free mark like the above or leave it.